requirements.txt

```diff
@@ -1,10 +1,10 @@
-blinker==1.6.2 ; python_version >= "3.8" and python_version < "4.0" \
-    --hash=sha256:4afd3de66ef3a9f8067559fb7a1cbe555c17dcbe15971b05d1b625c3e7abe213 \
-    --hash=sha256:c3d739772abb7bc2860abf5f2ec284223d9ad5c76da018234f6f50d6f31ab1f0
-certifi==2023.7.22 ; python_version >= "3.7" and python_version < "4.0" \
-    --hash=sha256:539cc1d13202e33ca466e88b2807e29f4c13049d6d87031a3c110744495cb082 \
-    --hash=sha256:92d6037539857d8206b8f6ae472e8b77db8058fec5937a1ef3f54304089edbb9
-cffi==1.15.1 ; python_version >= "3.7" and python_version < "4.0" \
+blinker==1.7.0 ; python_version >= "3.8" and python_version < "4.0" \
```

Comment:
why did this file change ? 
Wont this block us with customers ?

Reply:
I think it should be ok as its only "talk" about which package version to use for each python version but @omercnet will know better, @omercnet please double check me

Reply:
Related to pre-commit